falcon_kit/mains/ovlp_to_graph.py

```diff
@@ -818,16 +864,16 @@ def generate_string_graph(args):
             edge_data[ (v, w) ] = (rid, sp, tp, length, score, identity, type_)
             if w in sg.best_in:
                 nxsg.node[w]["best_in"] = v
+        elif (v, w) in chimer_edges:
+            type_ = "C"
         elif (v, w) in removed_edges:
             type_ = "R"
         elif (v, w) in spur_edges:
             type_ = "S"
         elif sg.e_reduce[(v, w)] == True:
             type_ = "TR"
 
-        line = '%s %s %s %5d %5d %5d %5.2f %s' %(
-                v, w, rid, sp, tp, score, identity, type_)
-        print >>out_f, line
+        print >>out_f, v, w, rid, sp, tp, score, identity, type_
```

Comment:
I really think it's better to use a format string. That way, we can see exactly what we'll get.